huaweicloud/services/dli/resource_huaweicloud_dli_database.go

```diff
@@ -85,10 +88,10 @@ func ResourceDliSqlDatabaseV1Create(ctx context.Context, d *schema.ResourceData,
 	// in the READ method.
 	d.SetId(dbName)
 
-	return ResourceDliSqlDatabaseV1Read(ctx, d, meta)
+	return ResourceDliSQLDatabaseV1Read(ctx, d, meta)
 }
 
-func GetDliSqlDatabaseByName(c *golangsdk.ServiceClient, dbName string) (databases.Database, error) {
+func GetDliSQLDatabaseByName(c *golangsdk.ServiceClient, dbName string) (databases.Database, error) {
```

Comment:
if it is not used by other package, it should be changed to getDliSQLDatabaseByName